packages/components/package.json

```
@@ -52,7 +52,7 @@
     "ember-focus-trap": "^1.1.0",
     "ember-get-config": "^2.1.1",
     "ember-modifier": "^4.1.0",
-    "ember-power-select": "^8.2.0",
+    "ember-power-select": "^8.6.0",
```

we had two patch releases since we started the work on this one, so we should bump this to latest 8.6.2
```suggestion
    "ember-power-select": "^8.6.2",
```

---

of course, this would require a small adjustment to the changelog entry and a `yarn.lock` update

---

we have a version bump in https://github.com/hashicorp/design-system/pull/2625#pullrequestreview-2534156160 as well, so you can get this in as is.

---

So is this one ok to just merge now as is?

---

Yes.
However, we're having issues with getting it approved. They automatically get dismissed after the `main` was merged into it. Would you be able to force-push from local to see if we can overcome this?